examples/cookbook/utils/verify-signature.js

```diff
@@ -0,0 +1,37 @@
+const { connect, keyStores } = require("near-api-js");
+const path = require("path");
+const homedir = require("os").homedir();
+const nacl = require("tweetnacl");
+const sha256 = require("js-sha256");
+
+const ACCOUNT_ID = "near-example.testnet";
+const CREDENTIALS_DIR = ".near-credentials";
+
+const credentialsPath = path.join(homedir, CREDENTIALS_DIR);
+const keyStore = new keyStores.UnencryptedFileSystemKeyStore(credentialsPath);
+
+const config = {
+    keyStore,
+    networkId: "testnet",
+    nodeUrl: "https://rpc.testnet.near.org",
+};
+
+verifySignature();
+
+async function verifySignature() {
+    const near = await connect({ ...config, keyStore });
+    const account = await near.account(ACCOUNT_ID);
+    const tokenMessage = new TextEncoder().encode(`Hello world, ${ACCOUNT_ID}`);
+    const signatureData = await account.connection.signer.signMessage(
+        tokenMessage,
+        ACCOUNT_ID,
+        "testnet"
```

Comment:
I think we should be referencing `config.networkId`, that way when if someone changes the example to try something against `mainnet`, they don't need to find and update this magic string for it to match their connection configuration :)